### PR TITLE
deploy disclaimer

### PR DESCRIPTION
**Pull request by LindsayYoung**
forewarned is forearmed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,7 +130,11 @@ Compile Sass as changes are made:
 
 ### Deployment
 
-*Likely only useful for 18F team members*
+*Likely only useful for 18F FEC team members*
+
+We use Travis for automated deploys after tests pass. If you want to deploy something it is much better to push an empty commit with a tag than doing a manual deploy.
```
Comment (ccostino) [inline README.md]: Should we include a line as an example, e.g.:

```bash
git checkout master
git commit --allow-empty -m 'Message indicating what the commit is for'
git tag -a -m 'Message about the tag' <tagname> <commit id generated above with commit above>
git push --follow-tags origin master
```

Does that look useful and right?

Comment (noahmanger) [inline README.md]: I like adding this example.

Comment (ccostino) [inline README.md]: @LindsayYoung, looks like this is still outstanding and we should probably get it merged - would adding this example be okay?

Comment (LindsayYoung) [inline README.md]: Yes looks good

**Pull request merged**